File: OrderNet-Server/src/main/java/bitc/fullstack503/ordernetserver/dto/app/WHOrderAppItemDTO.java
package bitc.fullstack503.ordernetserver.dto.app;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.math.BigDecimal;
import java.util.Date;

@Data
public class WHOrderAppItemDTO {

  @JsonProperty("order_id")
  private String orderId;

  @JsonProperty("order_item_id")
  private int orderItemId;

  @JsonProperty("part_id")
  private String partId;

  @JsonProperty("part_name")
  private String partName;

  @JsonProperty("part_cate")
  private String partCate;

  @JsonProperty("part_img")
  private String partImg;

  @JsonProperty("order_item_quantity")
  private int orderItemQuantity;

  @JsonProperty("order_item_price")
  private BigDecimal orderItemPrice;

  @JsonProperty("order_item_status")
  private String orderItemStatus;

  @JsonProperty("order_date")
  private Date orderDate;

  @JsonProperty("order_due_date")
  private Date orderDueDate;

  @JsonProperty("order_status")
  private String orderStatus;

  @JsonProperty("order_price")
  private BigDecimal orderPrice;

  @JsonProperty("branch_name")
  private String branchName;

  @JsonProperty("warehouse_stock")
  private Integer warehouseStock;
}

File: OrderNet-APP/app/src/main/java/bitc/fullstack/app/dto/WHOrderAppItemDTO.kt
package bitc.fullstack.app.dto

import com.google.gson.annotations.SerializedName
import java.math.BigDecimal
import java.util.Date

data class WHOrderAppItemDTO(

    @SerializedName("order_id")
    var orderId: String,

    @SerializedName("order_item_id")
    var orderItemId: Int,

    @SerializedName("part_id")
    var partId: String,

    @SerializedName("part_name")
    var partName: String,

    @SerializedName("part_cate")
    var partCate: String,

    @SerializedName("part_img")
    var partImg: String,

    @SerializedName("order_item_quantity")
    var orderItemQuantity: Int,

    @SerializedName("order_item_price")
    var orderItemPrice: BigDecimal,

    @SerializedName("order_item_status")
    var orderItemStatus: String,

    @SerializedName("order_date")
    var orderDate: Date,

    @SerializedName("order_due_date")
    var orderDueDate: Date,

    @SerializedName("order_status")
    var orderStatus: String,

    @SerializedName("order_price")
    var orderPrice: BigDecimal,

    @SerializedName("branch_name")
    var branchName: String,


    )
